components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/FileSelectionTable.java

```diff
@@ -358,7 +358,7 @@ private void removeSelectedFiles()
 	    int[] rows = table.getSelectedRows();
 	    if (rows == null || rows.length == 0) return;
 	    DefaultTableModel dtm = (DefaultTableModel) table.getModel();
-	    Vector<?> v = dtm.getDataVector();
+	    Vector v = dtm.getDataVector();
```

Comment:
This is really strange; I don't see how it makes sense. I can reproduce it in the Insight build but not with simpler test cases. Maybe it's a bug in the compiler; we can revisit it at a later date without blocking this PR.